includes/Init.php

```diff
@@ -54,9 +56,10 @@ public function staff_meta() {
 		}
 
 		$details = [
-			'name'  => get_the_title(),
-			'id'    => get_the_ID(),
-			'email' => base64_encode( get_post_meta( get_the_ID(), 'email', true ) ),
+			'name'       => get_the_title(),
+			'id'         => get_the_ID(),
+			'email' 		 => base64_encode( get_post_meta( get_the_ID(), 'email', true ) ),
+			'hide_email' => Settings::get( 'show_staff_email' ) == 'off'
```

Comment:
Add default value to Settings::get (should be the second parameter)

Reply:
I didn't end up using that parameter, so I've removed it